tests/snap_test.go

```diff
@@ -122,32 +92,49 @@ func setup() (teardown func(), err error) {
 	return
 }
 
-func waitForAppMessage(t *testing.T, appLogPath, expectedLog string, since time.Time) {
+func startAllClustersApp(t *testing.T) {
+	// remove existing temp files
+	utils.Exec(t, "rm -fr /tmp/chip_*")
+
+	logFile, err := os.Create(allClustersAppLog)
+	if err != nil {
+		t.Fatalf("Error creating log file: %s", err)
+	}
+
+	ctx, cancel := context.WithCancel(context.Background())
+	t.Cleanup(cancel)
+
+	cmd := exec.CommandContext(ctx, allClustersAppBin)
+	cmd.Stdout = logFile
+	cmd.Stderr = logFile
+
+	if err := cmd.Start(); err != nil {
+		t.Fatalf("Error starting application: %s", err)
+	}
+
+	t.Cleanup(func() {
+		utils.Exec(t, "rm -f /tmp/chip_*")
+	})
+}
+
+func waitForLogMessage(t *testing.T, logPath, expectedMsg string, since time.Time) {
```

Comment:
To distinguish this function from `utils.WaitForLogMessage()`:

```suggestion
func waitForLocalLogFileMessage(t *testing.T, logPath, expectedMsg string, since time.Time) {
```